package.json

```diff
@@ -39,6 +43,7 @@
     "@types/yargs": "^17.0.4",
     "@typescript-eslint/eslint-plugin": "^5.1.0",
     "@typescript-eslint/parser": "^5.1.0",
-    "eslint": "^8.0.1"
+    "eslint": "^8.0.1",
```

Comment:
let's add an eslint config file

Reply:
We already have one [here](https://github.com/OffchainLabs/arb-token-lists/blob/master/.eslintrc.js), we might want to enforce it during CI, what do you think?

Reply:
ah yes, you're right; enforcing it would be great 👍